### ISSUE TITLE
Is the smaller the loss the better,or the loss close to 0 the better?

### ISSUE THREAD

**Issue by AnsonDrew7** [shuaishiliu/SGCN#11]
For example,loss =-0.002 and loss =-0.033,which better?

**Comment by shuaishiliu** [comment]
> For example,loss =-0.002 and loss =-0.033,which better?

Hi,
the negative log likelihood loss function will be negative when the pdf is greater than 1. So, the more small loss more better, but be careful overfitting.
